Build fails for plugins because of SonarRunner

As one can see in [that buildlog](https://travis-ci.org/syncany/syncany-plugin-dropbox/builds/53163161), all plugin builds are failing because of SonarRunner.

```
FAILURE: Build failed with an exception.
* Where:
Script '/home/travis/build/syncany/syncany-plugin-dropbox/core/gradle/gradle/reports.sonarqube.gradle' line: 8
* What went wrong:
A problem occurred evaluating script.
> Failed to apply plugin [id 'sonar-runner']
   > Cannot add extension with name 'sonarRunner', as there is an extension already registered with that name.
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

see syncany/syncany-plugin-dropbox#3

--- a/gradle/gradle/plugins.settings.gradle
+++ b/gradle/gradle/plugins.settings.gradle
@@ -13,12 +13,17 @@
 
 rootProject.projectDir = file('../../../core')
 
-include 'syncany'
+
+//Not needed and breaks sonar
+//include 'syncany'
+
 include 'syncany-lib'
 include 'syncany-cli'
 include 'syncany-util'
 
-project(':syncany').projectDir = file('../../../core')
+//Not needed and breaks sonar
+//project(':syncany').projectDir = file('../../../core') 
+
 project(':syncany-lib').projectDir = file('../../../core/syncany-lib')
 project(':syncany-cli').projectDir = file('../../../core/syncany-cli')
 project(':syncany-util').projectDir = file('../../../core/syncany-util')

